syntax = "proto2";

// just cpp using LITE_RUNTIME
option optimize_for = LITE_RUNTIME;

package flremoting.protocol;

enum MsgType {
  STARTREMOTEINFOREQ = 10001;
  STARTREMOTEINFORESP = 10002;
  LIVEOUTPUT_START = 10003;
  LIVEOUTPUT_STOP = 10004;
  LIVEOUTPUT_FRAME_VIDEO = 10005;
  LIVEOUTPUT_FRAME_AUDIO = 10006;
  OBS_SCENEITEM_INFO = 10007;
  PREVIEWWINDOWSZINFO = 10008;
  PREVIEWCANVASMODECHANGE = 10009;
  OBSITEMINFO = 10010;
  OBSTEXTSOURCE = 10011;
  OBSIMAGESOURCE = 10012;
  OBSVIDEOSOURCE = 10013;
  SETINPUTVOLUME = 10014;
  GETINPUTVOLUMEREQ = 10015;
  GETINPUTVOLUMERESP = 10016;
  SETOUTPUTVOLUME = 10017;
  GETOUTPUTVOLUMEREQ = 10018;
  GETOUTPUTVOLUMERESP = 10019;
  OBSITEMEDITINFOREQ = 10020;
  OBSITEMEDITINFORESP = 10021;
  OBSDEVICEINFO = 10022;
  SETDEVICEINFOREQ = 10023;
  SETDEVICEINFORESP = 10024;
  SETALLITEMUNSELECT = 10025;
  MOUSEMENUEVENT = 10026;
  PHONECASTINGINFO = 10027;
  PREVIEWSZCHANGE = 10028;
  OBSCAMERAVIDEOBEAUTIFYREQ = 10029;
  OBSCAMERAVIDEOBEAUTIFYRESP = 10030;
  OBSCURSORINFO = 10031;
  OBSFPSINFO = 10032;
  INPUTVOLUMEINFO = 10033;
  OUTPUTVOLUMEINFO = 10034;
  UPDATEINPUTINFOREQ = 10035;
  UPDATEINPUTINFORESP = 10036;
  UPDATEOUTPUTINFOREQ = 10037;
  UPDATEOUTPUTINFORESP = 10038;
  MEANRATEINFO = 10039;
  OBSITEMEDIT = 10040;
  REMOTEESTABLISH = 10041;
}
